common/app/conf/AudioFlagship.scala

```diff
@@ -0,0 +1,9 @@
+package conf
+
+object AudioFlagship {
+  val tagName: String = "news/series/todayinfocus"
+  val title: String = "Today in Focus"
+  val seriesId: String = "today-in-focus"
```

Comment:
what is `seriesId` used for?

Reply:
It is currently not, but should be used to generate classnames

Reply:
I'll add it into the series header markup as a classname. Depends how resusable we want this to be